app/models/order_status.rb

```diff
@@ -16,34 +16,29 @@ class OrderStatus < ActiveRecord::Base
   end
 
   scope :for_facility, ->(facility) { where(facility_id: [nil, facility.id]).order(:lft) }
-  scope :new_os, -> { where(name: "New").limit(1) }
-  scope :inprocess, -> { where(name: "In Process").limit(1) }
-  scope :canceled, -> { where(name: "Canceled").limit(1) }
-  scope :complete, -> { where(name: "Complete").limit(1) }
-  scope :reconciled, -> { where(name: "Reconciled").limit(1) }
 
   def self.new_status
-    new_os.first
+    find_by(name: "New")
   end
 
   def self.complete_status
```

Comment:
To clarify--do you mean that you want all the `_status` methods to lose the status, basically? So it would become `OrderStatus.complete`, `OrderStatus.new`, etc?

I'm down to make that change. Do we have a lot of fork-specific code that we are concerned about migrating? Might be better to do a two stage migration if we want to make that change, in order to avoid possible collisions with the scopes that I eliminated in this PR--I'd rather get a method missing error than have some possible weird side effects.